src/utils.rs

```diff
@@ -117,15 +117,24 @@ pub(crate) async fn get_listen_ips(
     configuration.user_agent = Some(version());
     configuration.api_key = Some(api_key);
 
-    let listen =
-        zerotier_one_api::apis::network_api::get_network(&configuration, network_id).await?;
-    if let Some(assigned) = listen.assigned_addresses {
-        if assigned.len() > 0 {
-            return Ok(assigned);
+    match zerotier_one_api::apis::network_api::get_network(&configuration, network_id).await {
+        Err(error) => {
+            match error {
+                zerotier_one_api::apis::Error::ResponseError(_) => Err(anyhow!("Are you joined to {}?", network_id)),
```

Comment:
we should probably be checking the status codes here

Reply:
Yeah I agree. I'll take another stab at it. 
403 unauthorized could use it's own nice message.